### ISSUE TITLE
Dark mode does not persist after app close

### ISSUE THREAD

**Issue by hdlineage** [michaelkourlas/voipms-sms-client#206]
I'm using LG V30 on android 9.
Dark mode works the first time i toggle the option, however after closing the app completely, the app went back to light mode. 
I have to toggle the option again to get it back to dark mode.

**Comment by michaelkourlas** [comment]
Unfortunately, device specific issues like this are basically impossible for me to debug.

If you can explain what the problem is and suggest a fix, please feel free to reopen.

**Comment by smnc** [comment]
I actually have this issue as well, on two different devices: my Acer Chromebook Tab10, and my Samsung Galaxy Note 8.0; so it's not a device specific issue.

I manually set the app to use the "Dark" theme, and it work. But every time I close and reopen the app, while the setting itself persists, but the the actual theming doesn't. 
Changing the setting to "Light" and then back to "Dark" works, but every time I close and re-open the app, it defaults back to light.

**Comment by smnc** [comment]
![Screenshot_20200613-120922](https://user-images.githubusercontent.com/19196582/84573674-fa44a800-ad6f-11ea-86af-e73e3c920031.png)

**Comment by michaelkourlas** [comment]
Okay, that's interesting. I'll investigate this further.

**Comment by smnc** [comment]
I've got an update on this issue. 
My N5120 is running a custom Android 7.1 ROM without native support for system-wide dark mode. 
I've recently started using the "Dark Mode" app, by Julian Eggers, which seems to trick many apps in to thinking the system-wide dark mode is enabled. 
Since doing this, VOIP.ms SMS has persiststantly stayed in dark mode. 

I tried the same thing on my Chromebook Tab 10 with the same result.
